SemSur/.htaccess

```
@@ -0,0 +1,27 @@
+    
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+
+RewriteEngine on
+
+
+#Rewrite rules for semsur vocabulary 
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ http://sda.tech/SemSur/Documentation/Semsur.html  [R=303,NE,L]
+RewriteCond %{HTTP_ACCEPT} ^.applicationrdf+xml. 
```

That pattern looks wrong.  Perhaps this would work:
```suggestion
RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.*
```